src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -0,0 +1,346 @@
+Module Program
+    Sub Main(args As String())
+        Dim i1 As Integer'BIND:"Dim i1 As Integer"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MultipleVariableDeclarations()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim i1 As Integer, i2 As Integer, b1 As Boolean'BIND:"Dim i1 As Integer, i2 As Integer, b1 As Boolean"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (3 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+  IVariableDeclaration: b1 As System.Boolean (OperationKind.VariableDeclaration)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub DimAsNew()
+            Dim source = <![CDATA[
+Module Program
+    Class C
+    End Class
+    Sub Main(args As String())
+        Dim p1 As New C'BIND:"Dim p1 As New C"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: p1 As Program.C (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MultipleDimAsNew()
+            Dim source = <![CDATA[
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer'BIND:"Dim i1, i2 As New Integer"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
```

Comment:
>Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32) [](start = 4, length = 139)

Are both initializers the same node or distinct nodes? It would be good to assert that.